src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -193,13 +193,22 @@ AndroidJavaObject CreateNativeConfig(IConfiguration config)
                 obj.Call("setEndpoints", endpointConfig);
             }
 
-
-            // set version/context/maxbreadcrumbs/AppType/maxPersistedEvents/PersistUser/SendThreads
+            // set version/context/maxbreadcrumbs/AppType/maxPersistedEvents/PersistUser/versionCode
```

Comment:
You can probably just lose this comment rather than continually add to it (or change it to something generic that doesn't need changing).